core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -85,32 +120,31 @@ module.exports = class baseUser {
 
   /**
    * This function signs up a new user with the given username and email.
-   * @param {string} userName - The username of the user.
-   * @param {string} signInEmail - The email of the user.
    */
-  async signUpNewUser(userName, signInEmail) {
+  async signUpNewUser(userName: string, signInEmail: string): Promise<void> {
```

Comment:
Tangential, but since we're kinda doing cleanup, let's call the second arg "email", since "sign in" is a verb and we try not to start variables with verbs.